flutter_app/test/widget/display_all_logs_widget_test.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
+import 'package:imacs/screens/log_displayer_screen.dart';
+import 'package:imacs/widgets/display_all_logs_widget.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group(
+    'Display All Logs Widget',
+    () {
+      testWidgets(
+        'Displays airside logs by showing their respective path',
+        (WidgetTester tester) async {
+          const String pathToDirectory = 'C:\\Users\\emmao\\Documents\\logs';
```

Comment:
The CI/CD pipeline will fail with the current path, try creating a folder with test logs and pushing that as well. It's not that elegant, but it works (also use unix directory structure: ./folder/subfolder/etc.)